modules/osdk-crds.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// * applications/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-crds_{context}"]
+= Understanding your Custom Resource Definitions (CRDs)
+
+There are two types of Custom Resource Definitions (CRDs) that your Operator may
+use: ones that are _owned_ by it and ones that it depends on, which are
+_required_.
+
+[id="osdk-crds-owned_{context}"]
+== Owned CRDs
+
+The CRDs owned by your Operator are the most important part of your CSV. This
+establishes the link between your Operator and the required RBAC rules,
+dependency management, and other Kubernetes concepts.
+
+It is common for your Operator to use multiple CRDs to link together concepts,
+such as top-level database configuration in one object and a representation of
+ReplicaSets in another. Each one should be listed out in the CSV file.
+
+.Owned CRD fields
```

Comment:
@ecordell Do we need to point out the `must` or `optional` fields? For example, [etcd csv](https://github.com/operator-framework/community-operators/blob/master/community-operators/etcd/etcdoperator.v0.9.4-clusterwide.clusterserviceversion.yaml), there are no `Group` and `ActionDescriptors` fields.

Reply:
@ecordell what do you think?

Reply:
That makes sense to me; though the fields are defined required/optional in the schema for the api.

Reply:
Updated Tables 3, 4, 5, and 6 with a `Required/Optional` column, based on what was listed as `required` for `customresourcedefinitions` and `apiservicedefinitions` in https://github.com/operator-framework/operator-lifecycle-manager/blob/master/manifests/0000_50_olm_03-clusterserviceversion.crd.yaml.